README.md

```diff
@@ -3,18 +3,23 @@
         <img width="20%" src="/poweredbypysat.png" alt="pysat" title="pysat"</img>
 </div>
 
-# pysatMissionPlanning
-[![Build Status](https://travis-ci.org/pysat/pysatMissionPlanning.svg?branch=master)](https://travis-ci.org/pysat/pysatMissionPlanning)
-[![Coverage Status](https://coveralls.io/repos/github/pysat/pysatMissionPlanning/badge.svg?branch=master)](https://coveralls.io/github/pysat/pysatMissionPlanning?branch=master)
+# pysatMissions
+[![Documentation Status](https://readthedocs.org/projects/pysatMissions/badge/?version=latest)](http://pysatMissions.readthedocs.io/en/latest/?badge=latest)
 [![DOI](https://zenodo.org/badge/209358908.svg)](https://zenodo.org/badge/latestdoi/209358908)
 
-pysatMissionPlanning allows users to run build simulated satellites for TLE info and add empirical data.  It includes the pysat_sgp4 instrument module which can be imported into pysat.
+[![Build Status](https://travis-ci.org/pysat/pysatMissions.svg?branch=master)](https://travis-ci.org/pysat/pysatMissions)
+[![Coverage Status](https://coveralls.io/repos/github/pysat/pysatMissions/badge.svg?branch=master)](https://coveralls.io/github/pysat/pysatMissions?branch=master)
+[![Maintainability](https://api.codeclimate.com/v1/badges/83011911691b9d2076e9/maintainability)](https://codeclimate.com/github/pysat/pysatMissions/maintainability)
+
+pysatMissions allows users to run build simulated satellites for Two-Line Elements (TLE) and add empirical data.  It includes the pysat_ephem and pysat_sgp4 instrument modules which can be imported into pysat.
 
 Main Features
 -------------
 - Simulate satellite orbits from TLEs and add data from empirical models
-- Import ionosphere and thermosphere values through pyglow
-- Import coordidnates through apexpy
+- Import ionosphere and thermosphere model values through pyglow
+- Import magnetic coordinates through apexpy
+- Import magnetic coordinates through aacgmv2
+- Import geomagnetic basis vectors through pysatMagVect
```

Comment:
new name, OMMBV

Reply:
Looks like some updates are still needed with the name change. Since we still have pip-installable pysatMagVect, can we leave this until 0.3?  Probably will push that through this summer.